test/mocha.opts

```diff
@@ -1,2 +1,2 @@
 --recursive
--t 30000
+-t 60000
```

Comment:
so tests got slower with localstack? 😢

Reply:
@msluther its just for defensiveness so we can start the tests concurrently with starting localstack